packages/soundux/soundux.pacscript

```diff
@@ -2,7 +2,7 @@ name="soundux"
 version="0.2.7"
 url="https://github.com/Soundux/Soundux/releases/download/0.2.7/soundux-0.2.7.tar.gz"
 build_depends="build-essential cmake libx11-dev libxi-dev libwebkit2gtk-4.0-dev libappindicator3-dev libssl-dev libpulse-dev"
```

Comment:
```suggestion
build_depends="cmake libx11-dev libxi-dev libwebkit2gtk-4.0-dev libappindicator3-dev libssl-dev libpulse-dev"
```
`build-essential` is obsolete too, as it is a pacstall dependency, no need to add it separately

Citation
https://github.com/pacstall/pacstall/blob/35b3f057d4104fa6ded75e8faecc923e1ee8477d/install.sh#L91

Reply:
Thanks!